spark/src/main/java/org/apache/accumulo/spark/CopyPlus5K.java

```diff
@@ -76,8 +76,16 @@ private static void cleanupAndCreateTables(Properties props) throws Exception {
         client.tableOperations().delete(outputTable);
       }
       // Create tables
-      client.tableOperations().create(inputTable);
-      client.tableOperations().create(outputTable);
+      try {
+        client.tableOperations().create(inputTable);
+      } catch (TableExistsException e) {
+        // ignore
```

Comment:
Does this not represent an actual error condition - this implies that the proceeding table delete failed?  It shouldn't happen, but if it does then it seems something needs investigation.